ignition/azure/master.yaml.tmpl

```diff
@@ -1460,6 +1463,71 @@ storage:
               user: proxy
             name: service-account-context
           current-context: service-account-context
+    - path: /etc/kubernetes/config/scheduler-config.yaml
```

Comment:
Why we don't have kubeproxy config for Azure? Files for AWS and Azure should be almost the same imo, except calico.